futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -340,12 +347,91 @@ def _extract_exam_scores(representation_item: dict[str, Any]) -> None:
                 representation_item[possible_key] = score[1] if score else 'no attempt'
 
         representation = super().to_representation(instance)
-
```

Comment:
please recover the deleted empty line, here and everywhere unless you intentionally want to delete it

For this one, it is more readable to have the empty lines separating the logic